### ISSUE TITLE
Cannot build using .NET core SDK 2.1.3

### ISSUE THREAD

**Issue by flohgh** [tewtal/itemrandomizerweb#30]
I tried to build and run the project in order to test the new full item randomization feature. So I installed dotnet Core SDK 2.1.3, ran dotnet restore and dotnet run at repository root.

restore seemed to run fine but run told me that build failed. I tried dotnet build and got the following error : 
```
Microsoft (R) Build Engine version 15.3.409.57025 pour .NET Core
Copyright (C) Microsoft Corporation. Tous droits réservés.

C:\Users\flo\.nuget\packages\fsharp.net.sdk\1.0.5\build\FSharp.NET.Core.Sdk.targets(170,9): error MSB6006: Arrêt de "dotnet.exe" avec le code -2147450749. [C:\cygwin64\home\flo\src\itemrandomizerweb\ItemRandomizer\ItemRandomizer.fsproj]

ÉCHEC de la build.

C:\Users\flo\.nuget\packages\fsharp.net.sdk\1.0.5\build\FSharp.NET.Core.Sdk.targets(170,9): error MSB6006: Arrêt de "dotnet.exe" avec le code -2147450749. [C:\cygwin64\home\flo\src\itemrandomizerweb\ItemRandomizer\ItemRandomizer.fsproj]
    0 Avertissement(s)
    1 Erreur(s)

Temps écoulé 00:00:01.50
```
Sorry it is in French but basically it fails badly with error codes MSB6006/-2147450749. Googling these error codes gave no interesting results, so they don't seem that common.

I tried on a Windows 10/x64 and a Windows 7/x64 system, with the same result. I also tried on a Debian 9 system, but dotnet command just segfaults there...

Any idea what that might be? Missing package...? Or maybe I need to install VS?

**Comment by JamesAZ0** [comment]
I ran into this today. I'm no programmer but a couple quick searches gave me the gist of the problem. Fsharp.net.sdk is depreciated in netcoresdk2 however is still referred to in the itemrandomizer.fsproj and itemrandomizerweb.fsproj files. Implementing the recommendations in the link below on the two fsproj files, I was able to build, run the site and randomize a ROM successfully.

Link: [https://github.com/dotnet/netcorecli-fsc/wiki/How-to-migrate-1.0-projects-to-2.0#changes-in-fsproj]
